pages/user/[userId]/chat/[chatRoomId]/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { Flex, Input, Text } from '@chakra-ui/react';
+import {
+  GetServerSideProps,
+  InferGetServerSidePropsType,
+  NextPage,
+} from 'next';
+import { useEffect, useState } from 'react';
+
+import { userAPI } from 'apis';
+import useStomp from 'hooks/useStomp';
+import { ChatMeesageResponseType } from 'types/chatMessages';
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  const { userId, chatRoomId } = context.query;
+  let userInfo = {};
+
+  try {
+    userInfo = (await userAPI.getUser(parseInt(userId as string, 10))).data;
+  } catch (error) {
+    console.error(error);
+  }
+
+  return {
+    props: {
+      userInfo,
+      chatRoomId: parseInt(chatRoomId as string, 10),
+    },
+  };
+};
+
+const ChatRoom: NextPage = ({
+  userInfo,
+  chatRoomId,
+}: InferGetServerSidePropsType<typeof getServerSideProps>) => {
+  const [messages, setMessages] = useState<ChatMeesageResponseType>([]);
+  const { connect, disConnect, publish } = useStomp({
+    chatRoomId,
+    userInfo,
+    setMessages,
+  });
+
+  useEffect(() => {
+    connect();
+    return () => disConnect();
+  }, []);
+
+  useEffect(() => {
+    getMessages();
+  }, []);
+
+  const getMessages = async () => {
+    const { data } = await userAPI.getChatMessagesByChatRoomId({
+      chatRoomId,
+      offset: 0,
+      limit: 10,
+    });
+
+    setMessages([...data.reverse(), ...messages]);
```

Comment:
```suggestion
    const chatMessages = (await userAPI.getChatMessagesByChatRoomId({
      chatRoomId,
      offset: 0,
      limit: 10,
    })).data;

    setMessages([...chatMessages.reverse(), ...messages]);
```
data의 뜻이 쉽게 다가오지않아서 한번 바꿔보았는데 어떻게 생각하시는 지 궁금합니다!

Reply:
너무 좋습니다. 반영하겠습니다.!